src/main/java/com/mongodb/hibernate/internal/extension/service/StandardServiceRegistryScopedState.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.mongodb.hibernate.internal.extension.service;
```

Comment:
Most of the program elements we have represent part of the extension. The `extension` in the package name does not make sense, though originally I thought it would make sense.